jest.config.js

```diff
@@ -25,5 +25,33 @@ const customJestConfig = {
   },
 }
 
+// Check if the environment variable is set to enable test trace,
+// Insert a reporter to generate a junit report to upload.
+// This won't count for the retry to avoid duplicated test being reported twice
+// - which means our test trace will report test results for the flaky test as failed without retry.
+const shouldEnableTestTrace =
```

Comment:
Do we want to track these on PRs or should these only be recorded on the main branch?

Reply:
Turbopack currently does collect on all branch, and it's easy to filter out in datadog. If we feel it's too much we can reduce it later but it was helpful to track PR's pipeline time to time for some cases.